MatrixAlgorithms/MatrixAlgorithms.cpp

```diff
@@ -213,6 +212,11 @@ void TEST_010_subtraction_2()
     }
     assert(isException);
 }
+std::vector<std::vector<double>> TEST_011_transpose()
+{
+    Matrix2d mat1 = {{2, 3, 4, 8}, {5, 6, 7, 9}};
+    return mat1.transpose();
+}
```

Comment:
Create a test case. This function doesn't test anything. Use 'assert' function to test you code.